storages/backends/gcloud.py

```diff
@@ -20,6 +21,7 @@
 class GoogleCloudFile(File):
     def __init__(self, name, mode, storage):
         self.name = name
+        self.mime_type = mimetypes.guess_type(name)[0]
```

Comment:
Now that I've looked more closely, I don't see a good place to add this. Django's storage API doesn't have any support for MIME types as far as I can tell, so we're well within our rights to set whatever we want when saving a file to the storage provider.